packages/eslint-plugin/src/rules/component-class-suffix.ts

```diff
@@ -10,15 +10,14 @@ type Options = [
 ];
 export type MessageIds = 'componentClassSuffix';
 export const RULE_NAME = 'component-class-suffix';
-
 const STYLE_GUIDE_LINK = 'https://angular.io/styleguide#style-02-03';
 
 export default createESLintRule<Options, MessageIds>({
   name: RULE_NAME,
   meta: {
     type: 'suggestion',
     docs: {
-      description: `Classes decorated with @Component must have suffix "Component" (or custom) in their name. See more at ${STYLE_GUIDE_LINK}.`,
+      description: `Component class names should end with one of these suffixes: {{suffixes}}. See more at ${STYLE_GUIDE_LINK}`,
```

Comment:
Did you mean to update the docs.description as well? This dynamic value `{{suffixes}}` won't be replaced here surely?

Reply:
Damn, a bad replace, sorry.